.gitmodules

```
@@ -10,3 +10,12 @@
 [submodule "lib/mbedtls"]
 	path = lib/mbedtls
 	url = https://github.com/ARMmbed/mbedtls
+[submodule "lib/mbed-cloud-client"]
+	path = lib/mbed-cloud-client
+	url = https://github.com/PelionIoT/mbed-cloud-client
+[submodule "lib/cpputest"]
+	path = lib/cpputest
+	url = git@github.com:cpputest/cpputest.git
+[submodule "lib/docopt.c"]
+	path = lib/docopt.c
+	url = git@github.com:docopt/docopt.c.git
```

please use https link for public repositories
```suggestion
	url = https://github.com/docopt/docopt.c.git
```